db/migrate/20230503083736_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index: true
```

Comment:
- [ ] Good job setting up data models, however, please notice as mentioned in the requirements 
![Screenshot from 2023-05-03 16-42-42](https://user-images.githubusercontent.com/97521557/235901084-625fb945-ab47-45c8-810b-a58c5872e9da.png)

we have to make sure the column name for referencing user should be author_id you can do that with `t.references :author, foreign_key: { to_table: 'users' }`

notice: you will have to create a new migration file for this